lms/static/sass/multicourse/_courses.scss

```diff
@@ -326,11 +326,24 @@ $facet-background-color: #007db8;
   .filters {
     @include clearfix();
     margin-top: ($baseline/2);
-    border-top: 2px solid $courseware-button-border-color;
-    border-bottom: 2px solid $courseware-button-border-color;
     width: 100%;
     height: auto;
     max-height: ($baseline*10);
+    overflow: hidden;
+
+    &.animate {
```

Comment:
one small nit here is that ideally we could have  classes names with clear state / case labeling, (ex: is-collapsed, is-expanding, is-active, etc.) In this case I dont know where .animate or .slide-up are necessarily being applied without reviewing the JS. https://smacss.com/book/type-state

aside: the transition when the filters are removed is a nice addition here, thanks for adding this!